node/clients/checkout.ts

```diff
@@ -0,0 +1,44 @@
+import type { InstanceOptions, IOContext, RequestConfig } from '@vtex/api'
+import { JanusClient } from '@vtex/api'
+
+import { statusToError } from '../utils/utils'
+
+export default class Checkout extends JanusClient {
+  constructor(ctx: IOContext, options?: InstanceOptions) {
+    super(ctx, {
+      ...options,
+      headers: {
+        ...(options?.headers ?? {}),
+        'Content-Type': 'application/json',
+        Accept: 'application/json',
+        'X-Vtex-Use-Https': 'true',
+      },
+    })
+  }
+
+  public getNearestPickupPoints = (lat: string, long: string) => {
+    return this.get<CheckoutOutput>(
+      this.routes.getNearestPickupPoints(lat, long),
+      {
+        metric: 'checkout-nearestPickupPoints',
+      }
```

Comment:
Maybe we could use `.then()` to directly return the `items` array, like we do [here](https://github.com/vtex-apps/cross-device-cart/blob/V1/node/clients/checkout/index.ts#L72)